### ISSUE TITLE
Breaking of import lines

### ISSUE THREAD

**Issue by sjrd** [olafurpg/scalafmt#101]
Original:
```scala
import scala.language.implicitConversions

import org.scalajs.dom.experimental.serviceworkers.{ServiceWorkerGlobalScope, ServiceWorkerRegistration}

import scala.scalajs.js
```

Scalafmt:
```scala
import scala.language.implicitConversions

import org.scalajs.dom.experimental.serviceworkers.{ServiceWorkerGlobalScope,
ServiceWorkerRegistration}

import scala.scalajs.js
```

I would like the input to look like this:
```scala
import scala.language.implicitConversions

import org.scalajs.dom.experimental.serviceworkers.{ServiceWorkerGlobalScope, ServiceWorkerRegistration}

import scala.scalajs.js
```
or
```scala
import scala.language.implicitConversions

import org.scalajs.dom.experimental.serviceworkers.{
  ServiceWorkerGlobalScope, ServiceWorkerRegistration
}

import scala.scalajs.js
```

Using
* 0.1.1
* default

**Comment by olafurpg** [comment]
I have a temporary fix that will keep each import statement on a single line. Produces this:
```
import org.scalajs.dom.experimental.serviceworkers.{ServiceWorkerGlobalScope, ServiceWorkerRegistration}
```

Am I free to close this issue? IMO, if your import statement is this long you should not wrap them in {}. I'm sure some people will disagree, and if they're motivated they can open an issue.

**Comment by sjrd** [comment]
Fine by me.

**Comment by regadas** [comment]
what about the following? sometimes imports can be pretty long.

```scala
import scala.language.implicitConversions

import org.scalajs.dom.experimental.serviceworkers.{
  ServiceWorkerGlobalScope,
  ServiceWorkerRegistration
}

import scala.scalajs.js
```

**Comment by olafurpg** [comment]
I agree that is a nice fallback layout if the imports don't fit on one line. Adding this should not be a big hassle. I'll reopen this issue, although I can't promise when I'll have time to get to it.

**Comment by pascoej** [comment]
@regadas 's syntax is the holy grail.

**Comment by olafurpg** [comment]
Closed by 5a40e1451f95a63d1021389788145bbff5b14956

**Comment by regadas** [comment]
@olafurpg nice! 👍
